src/dnx.coreclr/dnx.coreclr.cpp

```diff
@@ -332,6 +349,9 @@ extern "C" __declspec(dllexport) HRESULT __stdcall CallApplicationMain(PCALL_APP
     size_t cchTrustedPlatformAssemblies = 0;
     LPWSTR pwszTrustedPlatformAssemblies = nullptr;
 
+    // set handler for security-enhanced CRT functions
+    _set_invalid_parameter_handler(invalid_parameter_handler);
```

Comment:
Hum. It's not a bad idea to set a handler. However, the implementation of the function be passed in seems not to be inline with what expected. On [MSDN(https://msdn.microsoft.com/en-us/library/a9yf33zb.aspx)] it says 
> It should not return control to the main function unless you are confident that the error is recoverable.

My another concern is that replace the handler will change all the behavior down the roads, will this impact any existing error handling logic we have today? (Per discussion with @BrennanConroy the default handler will fast-fail and leave no trace. In that sense this could be better.)